_rules/heading-name-is-not-only-whitespace-ffd0e9.md

```diff
@@ -0,0 +1,185 @@
+---
+id: ffd0e9
+name: Heading name is not only whitespace
+rule_type: atomic
+description: | 
+  This rule checks that each heading does not have an accessible name that is only whitespace.
+
+success_criterion: 
+- 1.3.1 # Info and Relationships (A)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+- Kasper Isager
+---
+
+## Applicability
+
+This rule applies to any HTML element with the [semantic role](#semantic-role) of `heading` that is [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+## Expectation
+
+The [accessible name](#accessible-name) of the test target, if there is an accessible name, is not only [whitespace](#whitespace).
+
+**Note:** In the [Accessible Name and Description Computation](https://www.w3.org/TR/accname-1.1/#mapping_additional_nd_te) all carriage returns, newlines, tabs, and form-feeds are replaced with a single space.
+
+## Assumptions
+
+This rule assumes that having an element that unintentionally shows up programmatically to a user of assistive technologies as a heading element, but is not shown visually as a heading on the page, is a violation of WCAG Success Criterion 1.3.1 Info and Relationships.
+
+## Accessibility Support
+
+Handling of headings containing only whitespace characters, carriage returns, newlines, tabs, and form-feeds varies between different assistive technologies and browsers. This means that even though the outcome of this rule is *failed*, users of certain assistive technology and browser combinations might not experience an issue.
```

Comment:
align with current expectation. Different what is considered an empty heading...